File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Request input for Height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    for (int row = 1; row <= height; row++)
    {
        // Printing Space on Left Side
        for (int column = (height - row); column > 0; column--)
        {
            printf(" ");
        }

        // Printing # on Right Side
        for (int column = 1; column <= row; column++)
        {
            printf("#");

        }

        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int


def main():

    # Get Height from Standard Input
    height = 0
    while (height <= 0 or height > 8):
        height = get_int("Height: ")

    # Draw Pyramid
    for y in range(1, height + 1):

        # Draw Pyramid on left hand side
        print(" " * (height - y), end="")
        print("#" * y, end="")

        # Draw space on Center
        print("  ", end="")

        # Draw pyramid on right hand side
        print("#" * y)


if __name__ == "__main__":
    main()